### ISSUE TITLE
webTorrent

### ISSUE THREAD

**Issue by jimmywarting** [torrentsTime/embed#6]
Just saw this on TPB. Awesome!

What would be more awesome is if you used [webTorrent](https://github.com/feross/webtorrent) so ppl can share from p2p using WebRTC without installing any program

They are starting to implement WebRTC to desktop clients ([webtorrent#369](https://github.com/feross/webtorrent/issues/369)) to make it a Hybrid by supporting both WebRTC & Regular.

**Comment by robinpaulson** [comment]
I agree, webrtc is mature and stable enough now, plus the source for webtorrent is trivially available. In contrast, the plugins for this are platform-dependent and where is the code for the firefox and chrome plugins?

**Comment by tusharmath** [comment]
Webtorrent clients only connect to other webtorrent clients (unless you put a hybrid client somewhere) which basically are quite less. This will result in having less content on the network.

**Comment by jimmywarting** [comment]
WebTorrent is pushing to make uTorrent, Vuze, Libtorrent and everything else a hybrid client so a website can connect to everyone that isn't a browser. It just have to get one foot in to make it happen.
It's just an opt-in for people to use WebRTC protocol instead. So i don't believe the result will be less content on the network. (okey maybe a small %)

But I believe more in making it DEAD SIMPLE for people to start streaming a video just by visiting a website without installing any programs and making it OS independent. Then we have iOS-safari on board as well. (Apple don't allow any torrent apps in the store)
